.editorconfig

```
@@ -19,7 +19,7 @@ trim_trailing_whitespace = true
 # C# files
 [*.cs]
 
-file_header_template = Copyright © myCSharp 2020-2021, all rights reserved
+file_header_template = Copyright © myCSharp 2020-2022, all rights reserved
```

Do we need to specify the year at all?
So we have to update next year again...
Is 
```suggestion
file_header_template = Copyright © myCSharp, all rights reserved
```
sufficient? I see quite a lot other repos that don't specify the year too.